addons/mail/static/src/discuss/channel_invitation.js

```diff
@@ -45,6 +46,7 @@ export class ChannelInvitation extends Component {
             this.props.thread.id,
         ]);
         const Partners = results["partners"];
+        const thread = this.threadService.getThread(this.props.thread.id);
```

Comment:
This looks incorrect:
- `getThread(id)` instead of `getThread(model, id)`
- This is a method for chatter specifically, it should not be used for channels and chats
- `this.props.thread` is already a thread model, so `getThread()` looks useless